### ISSUE TITLE
Input Variable Missing from Correlations and Risk Tornado Diagram

### ISSUE THREAD

**Issue by vinunderwood** [pyscripter/XLRisk#15]
I created a workbook with 4 input variables.  I ran a simulation that appears to be successful.  However, on the Risk Results tab only the first 3 input variables are listed in the Correlations area and in the Risk Tornado Diagram.

The input variable missing is ''Final Calc'!$F$17 - it is a lognormal distribution.

The workbook is "Provenir Decision Engines - Queuing Perspective v3.0.xlsm" - I copied it to Google Docs.  Link is https://docs.google.com/spreadsheets/d/1Z2MI_9kisowW70XRjCVWTZLAcQ83bFGy/edit?usp=sharing&ouid=111926237364535382821&rtpof=true&sd=true.

Feel free to download a local copy to look at it.  Please let me know when you have done so - I would like to disable this link.

Thanks,
Vin Underwood

**Comment by pyscripter** [comment]
I tried the link but access was denied.

**Comment by vinunderwood** [comment]
oops.  I gave you a link before that worked.  Let me try again.

**Comment by vinunderwood** [comment]
ok, try this. https://docs.google.com/spreadsheets/d/1Z2MI_9kisowW70XRjCVWTZLAcQ83bFGy/edit?usp=sharing&ouid=111926237364535382821&rtpof=true&sd=true

It's the same link but I changed the sharing options.

**Comment by pyscripter** [comment]
Cell ''Final Calc'!$F$17" has no dependent  cells and it certainly does not affect the output cell ''Final Calc'!$F$19.   This is why the correlation is empty.

**Comment by vinunderwood** [comment]
I see it now.  Major logic error on the sheet "Final Calc".  thank you
